src/app/user/getAllUsers.js

```diff
@@ -1,5 +1,5 @@
 /**
-  * function for getter user.
+  * function for getter users.

```

Comment:
function to fetch users ?

Reply:
Yes, usesless for the Front end :)
It come from the boilerplate. 
Pherhaps To remove it.